src/doghouse/cli/main.py

```
@@ -0,0 +1,782 @@
+):
+    """Bundle PR history and metadata into a black box repro file."""
+    repo, repo_owner, repo_name, pr = resolve_repo_context(repo, pr)
+
+    storage = JSONLStorageAdapter()
+    snapshots = storage.list_snapshots(repo, pr)
+
+    github = GhCliAdapter(repo_owner=repo_owner, repo_name=repo_name)
+    metadata = github.get_pr_metadata(pr)
+
+    # Capture recent git log for context
+    git_log = subprocess.run(["git", "log", "-n", "10", "--oneline"], capture_output=True, text=True).stdout
```

Ruff (0.15.7)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)